app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -994,27 +987,19 @@ class MainActivity :
 
     override fun onMiniPlayerHidden() {
         val padding = binding.bottomNavigation.height
-        updateSnackbarPosition(padding)
+        updateSnackbarPosition(0)
         settings.updateBottomInset(0)
     }
 
     private fun updateSnackbarPosition(bottomPadding: Int) {
         binding.snackbarFragment.updateLayoutParams<ViewGroup.MarginLayoutParams> {
-            bottomMargin = bottomPadding
-        }
-    }
-
-    private fun setupSnackbarPosition() {
-        // we manually position the snackbar to account for the mini player and bottom navigation, consume the insets so extra padding isn't added to the snackbar
-        ViewCompat.setOnApplyWindowInsetsListener(binding.snackbarFragment) { _, _ ->
-            WindowInsetsCompat.CONSUMED
+            bottomMargin = bottomPadding + 16.dpToPx(binding.snackbarFragment.context)
```

Comment:
Is `16` just a simple space value or does it come from somewhere?

Reply:
It's just to add some padding to the snack bar, but I'm currently reworking its position as it isn't quite right. In landscape, it's slightly behind the mini player.

Reply:
|  |  |  |
| --- | --- | --- |
| ![Screenshot_20241129_221236](https://github.com/user-attachments/assets/fa8475a5-ae36-4f4a-b214-0961da8f93af) | ![Screenshot_20241129_221225](https://github.com/user-attachments/assets/23f75407-e8e0-4562-b68b-de6eed39bfe8) | ![Screenshot_20241129_221745](https://github.com/user-attachments/assets/321359f9-fc4d-4c3d-aa61-a55d8353ef74) |

I've improved the positioning and removed the 16.dp padding.